src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -30,7 +33,23 @@ export const useTradeRoutes = (
     state: { wallet },
   } = useWallet()
 
-  const [defaultSellAssetId, defaultBuyAssetId] = getDefaultPair()
+  // fixme: abstract to custom hook
```

Comment:
Leaving this here as we want to abstract this away and are starting to duplicating the logic: We had some discussion about the naming of the ChainReference part, which in the context of ETH JSON-RPC ChainId is currently referred to as `chainId` / `evmChainId`.
Going forward, we will want to consolidate the naming, most likely as `ethNetwork`  or `ethNetworkVersion`. That will make this (as an abstracted hook then) a bit less confusing, not referring to `chainId` at all.

Reply:
Sounds good - used "eth network" terminology in https://github.com/shapeshift/web/pull/2223/commits/12582d02b2c0494078c59fb7fb37cdf2cf3a3459.